recipe/meta.yaml

```diff
@@ -40,11 +39,13 @@ requirements:
     - libgomp      # [linux]
     - parallel-hashmap
     - cutlass                        # [cuda_compiler_version not in (undefined, 'None')]
+    - cuda-version =={{ cuda_compiler_version }}  # [cuda_compiler_version not in (undefined, 'None')]
```

Comment:
We can skip this in `requirements/build` as `{{ compiler('cuda') }}` pulls this in

```suggestion
```

With `requirements/host` we still need it as there isn't another good way to provide it automatically